docs/source/guides/logging.rst

```diff
@@ -2,94 +2,115 @@
+          # whether to log the content type of the response
           response_content_type: True
 
-In addition we provide the following parameters that can enabled or disabled in each log line. Each of these parameters comes from the http headers in the requests and response.
-
-- `request_content_length` is the size of the content that is being received
-- `request_content_type` is the type of content in the request
-- `response_content_length` is the content length of the data that is being returned in the response
-- `response_content_type` is the type of data being returned in the response
 
 Model Runner Request Logging
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+""""""""""""""""""""""""""""
 
-Depending on how you’ve configured BentoML, the webserver may be separated from the model runner. In either case, we have special logging that is enabled specifically on the model side of the request. You may configure the runner access logs under the runners parameter at the top level of the bentofile.yaml
+Depending on how you've configured BentoML, the webserver may be separated from the model runner.
+In either case, we have special logging that is enabled specifically on the model side of the
+request. You may configure the runner access logs under the runners parameter at the top level of
+your ``bentofile.yaml``
 
-.. code-block:: python
+.. code-block:: yaml
 
     runners:
         logging:
           access:
-              enabled: False
-              request_content_length: True
-              request_content_type: True
-              response_content_length: True
-              response_content_type: True
-
-Each additional parameter may be configured to be shown or not:
-
-- `request_content_length` is the size of the content that is being received coming from the web service
-- `request_content_type` is the type of content in the request coming from the web service
-- `response_content_length` is the content length of the data that is being returned in the response to the webservice
-- `response_content_type` is the type of data being returned in the response to the webservice
\ No newline at end of file
+              enabled: True
+              ...
+
+The available configuration options are identical to the webserver request logging options above.
+These logs are disabled by default in order to prevent double logging of requests.
+
+
+Library Logging
+---------------
+
+When using BentoML as a library, BentoML does not configure any logs. By default, Python will configure a root logger that logs at level WARNING and higher. If you want to see BentoML's DEBUG or INFO logs, register a log handler to the ``bentoml`` namespace:
+
+.. code-block:: python
+
+    import logging
+
+    ch = logging.StreamHandler()
+    ch.setLevel(DEBUG)
+
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    ch.setFormatter(formatter)
+
+    bentoml_logger = logging.getLogger("bentoml")
+    bentoml_logger.addHandler(ch)
```

Comment:
```suggestion
    import logging

    ch = logging.StreamHandler()
    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
    ch.setFormatter(formatter)

    bentoml_logger = logging.getLogger("bentoml")
    bentoml_logger.addHandler(ch)
    bentoml_logger.setLevel(logging.DEBUG)

```

Reply:
the sample code doesn't seem to work, calling setLevel on the bentoml logger fixed it for me